rasa/nlu/training_data/loading.py

```diff
@@ -51,7 +51,7 @@
 # ##
 # * intent/response_key
 #   - response_text
-_nlg_markdown_marker_regex = re.compile(r"##\s*.*\n\*.*\/.*\n\s*\t*\-.*")
+_nlg_markdown_marker_regex = re.compile(r"##\s*.*\n\*.*(?<!: )\/.*\n\s*\t*\-.*")
```

Comment:
I'm not a big regex expert, but I think this one would be a bit simpler:
```suggestion
_nlg_markdown_marker_regex = re.compile(r"##\s*.*\n\*[^:]*\/.*\n\s*\t*\-.*")
```

Reply:
Yeah that makes sense